components/goalsForm/goalsForm.tsx

```diff
@@ -1,4 +1,5 @@
 import React, {useState} from "react";
+import {themes as T} from "../../styles/theme";
```

Comment:
```suggestion
import {themes as T} from "styles/theme";
```
저희가 설정한 absolute path 를 이용해요!

Reply:
"styles/theme"으로 path를 적을 수 있는 이유가 무엇인가요?